appenders: [file, screen]
levels: [trace, info]
pattern_layout: "[%5p][%d{HH:mm:ss,SSS}][%M:%L][%c] %m%n"
file_name: "freq_est"
# REQUIRED:
carrier:
  angular_frequency: 0.5  # angular frequency rad/s
  amplitude: 0.1 #
  periods:   2.0  # number of repetitions of the carriers (round to ceil)
warmup_time: 10.0
input:
  max_pos: 0.05
  max_vel: 1.0
  max_acc: 3.0

  min_angular_frequency: 1.0
  max_angular_frequency: 100.0
  harmonics_number: 10

